cmd/misc/main.go

```diff
@@ -245,3 +258,91 @@ func ClearBigtable(family string, key string, dryRun bool, bt *db.Bigtable) {
 		logrus.Infof("%v keys have been deleted", len(deletedKeys))
 	}
 }
+
+func IndexOldEth1Blocks(startBlock uint64, endBlock uint64, batchSize uint64, concurrency uint64, transformerFlag string, bt *db.Bigtable) {
```

Comment:
This should include some sanity checks:
1) `startBlock` < `endBlock`
2) `batchSize` > 0
3) `concurrency` > 0
4) `bt` != nil

These checks should be done first i.e. even before `client, err := rpc.NewErigonClient(utils.Config.Eth1ErigonEndpoint)` is executed.This applies to https://github.com/gobitfly/eth2-beaconchain-explorer/blob/8f2dd48904c8628f64813ced7a612f5bbad8c0b7/cmd/misc/main.go#L270-L275 as well.

Reply:
added, except for batchSize that defaults maxes 1 - so it's optional